### PR TITLE
Manager Page

### PR DESCRIPTION
**Pull request by swilliams**
[Review after #54]

This adds details about the role of a Manager, how you get to be one, and what they do.

### CODE REVIEW

File: docs/Team/management.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+You should already have been at the Senior Developer level for a couple of years, probably even at Lead. If management is one of your career goals you should discuss it with your current manager. They can help you plan for that. When the time is right we will give you a supervisory role over several people and evaluate how that goes. This is a little different than the title of manager, in that you have direct reports, and will be expected to coach and grow them, but without the responsibility of recruiting and discipline.
+
+If that supervisory period goes well, and the timing and budget permits, you will be made an official manager. If it turns out the supervisory period doesn't work, that's ok, as stated, management is definitely not for everybody. In that case, you'll go back to your regular non-supervisory duties.
```
Comment (krantzinator) [inline docs/Team/management.md]: change comma to semicolon `that's ok, as stated` -> `that's ok; as stated`

File: docs/Team/index.md [hunk 2]
```diff
@@ -15,88 +15,25 @@ If you're reading this page, you probably want to know more about the actual dev
...
+### What title is not
+Title does not inflate anyone's worth in the company. We’re a relatively flat organization and that benefits us. A Lead Developer is not necessarily anyone’s boss, though they should be demonstrating leadership. In fact, everyone can (and should) be demonstrating leadership, regardless of title.
 
-Leads mentor more inexperienced developers more often. This could be answering questions, pair programming, or running code reviews. Leads help Software Developers grow to Seniors and Seniors to Leads.
+There is no pressure to gain ‘rank’ just for the sake of doing so. If you’re happy as a Software Developer, that’s great. Keep it up. However, it can be pretty fulfilling to add some more arrows to your quiver.
 
-They have more conversations with clients. Per the description, Leads will be presenting project plans to clients, getting their buy-in, and working with the PM team to keep them apprised every step of the way.
+All developers, regardless of title need to carry themselves well, have empathy for others, and demonstrate our Core Values.
```
Comment (mmislam101) [inline docs/Team/index.md]: comma after "regardless of title":
All developers, regardless of title, need to carry themselves well...

Comment (mmislam101) [reply docs/Team/index.md]: Maybe link to our core values web page?

Comment (swilliams) [reply docs/Team/index.md]: @mmislam101 yeah... the core values page got removed in the site refresh. I've been told that they'll come back at some point.

**Pull request merged**